src/Plutus/Certification/API/Routes.hs

```diff
@@ -446,8 +447,34 @@ instance ToSchema DAppBody where
     return $ NamedSchema (Just "DAppBody") $ profileSchema
         & properties . at "githubToken" ?~ apiGitHubAccessTokenSchema
 
-instance ToSchema ProfileBody
+-- | A phantom type for website for swagger schema
+data Website
 
+instance ToSchema Website where
+  declareNamedSchema _ = do
+    return $ NamedSchema (Just "BaseUrl") $ mempty
+      & type_ ?~ SwaggerString
+      & SL.pattern ?~ ghAccessTokenPattern
```

Comment:
Should be 
`$ SL.pattern ?~ baseUrlPattern`

Reply:
exactly 😔

Reply:
done